packages/services/storage/src/index.ts

```diff
@@ -77,10 +77,12 @@ export async function createStorage(connection: string): Promise<Storage> {
     return {
       id: user.id,
       email: user.email,
-      externalAuthUserId: user.external_auth_user_id,
-      provider: getProviderBasedOnExternalId(user.external_auth_user_id),
+      superTokensUserId: user.supertoken_user_id,
+      // TODO: figure out what to do here.
+      provider: getProviderBasedOnExternalId(user.external_auth_user_id ?? ''),
```

Comment:
I think we can still store the external_auth_user_id in our database when you sign up via supertokens.

Reply:
It is now stored in the access token payload + db upon signup